models/__init__.py

```diff
@@ -0,0 +1,2 @@
+from . import course
```

Comment:
Name files according to models:

```suggestion
from . import openacademy_course
```

Same below.

Reply:
Just to be sure: you mean compact all 10 commits of this PR into one?

Reply:
Yes